packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -519,7 +519,7 @@ const schema: SchemaType<"Users"> = {
     form: {
       options: function() {
         const groups = _.without(
-          _.keys(userGroups),
+          _.keys(getAllUserGroups),
```

Comment:
need to call this function

Reply:
Oof, good catch. Sad that the type system didn't get this one.